doc/decisions/meta-decisions/use-arc42-for-architecture-overview-documents/index.md

```diff
@@ -0,0 +1,62 @@
+# Adoption of Architecture Overview Documents
+
+**Status**: Proposed
+
+**Level**: 3
+
+**Updated**: 2025-07-24
+
+## Situation: Context and Problem Statement
```

Comment:
Can you align the headings with the [approved template](https://gig-cymru-nhs-wales.github.io/architecture-decision-records/design-authority/dhcw/architecture-decision-record-template/)